contrib/sendmail/vacation/vacation.1

```diff
@@ -206,7 +206,7 @@ reads the first line from the standard input for a
 UNIX
 ``From''
 line to determine the sender.
-Sendmail(8)
+.Xr sendmail(8)
```

Comment:
.Xr sendmail 8

Reply:
Okay, I can give that a try. Would it be preferred a pull request PER set of diffs or to have multiple diffs in one pull request?

Appreciate the input as this is first pull request and was trying to balance the set of changes.